changelog.config.js

```diff
@@ -0,0 +1,92 @@
+module.exports = {
+  disableEmoji: true,
+  list: [
+    "build",
+    "chore",
+    "ci",
+    "docs",
+    "feat",
+    "fix",
+    "perf",
+    "refactor",
+    "revert",
+    "style",
+    "test",
+  ],
+  maxMessageLength: 64,
+  minMessageLength: 3,
+  maxHeaderWidth: 72,
+  questions: [
+    "type",
+    "scope",
+    "subject",
+    "body",
+    "breaking",
+    "issues",
+    "lerna",
+  ],
+  scopes: [],
+  types: {
+    build: {
+      description:
+        "Changes that affect the build system or external dependencies (example scopes: gulp, broccoli, npm)",
+      emoji: "🏗",
```

Comment:
These all emojis on this file are very confusing to me.  Would you mind if you delete these emojis?

Reply:
@takeutak Done: https://github.com/hyperledger/cactus/pull/337/commits/30ad1b42c5a749b62f7eb26f74c6796d9e39e8f8